nature/models.py

```diff
@@ -625,7 +625,7 @@ class Meta:
         db_table = 'tapahtuma'
 
     def __str__(self):
-        return str(self.register_id)
+        return 'Transaction #{0}'.format(self.id)
```

Comment:
If this is something that should be shown in the UI then it should be wrappen in `_()`